src/components/Input/Input.spec.js

```diff
@@ -43,7 +80,7 @@ describe('Input', () => {
   });
 
   it('should prioritize error over optional styles', () => {
```

Comment:
Maybe there was a mixed intention here to write a test that makes disabled inputs never render as invalid? The test you fixed also makes sense.

Reply:
I'll add more tests for Inputs with disabled & invalid/hasWarning.